visualizer/index.html

```diff
@@ -253,6 +253,7 @@
       padding: 0;
     }
     .contextMenu li {
+      position: relative;
```

Comment:
I added submenu to the `Force Evaluation`, this is used for aligning the submenu (in the `styl/semanticsEditor.css`)